### PR TITLE
[DCOS-58439][DS] Don't fail run.sh if previous infra components already installed

### PR DESCRIPTION
**Pull request by alexeygorobets**
## What changes were proposed in this pull request?

Resolves [DCOS-58439](https://jira.mesosphere.com/browse/DCOS-58439)

## How were these changes tested?

I have tested it manually: 
1) Throw an error between  Zookeeper and Kafka installations
2) Check if output `infrastructure.json` only contains Zookeeper
2) Rerun script without throwing an error
3) Check if output `infrastructure.json` contains all services (Zookeeper, Kafka, Cassandra)

### CODE REVIEW

File: scale-tests/setup_streaming.py [hunk 1]
```diff
@@ -75,6 +76,11 @@ def install_package(package_name: str,
 
     service_options = scale_tests_utils.get_service_options(service_name, service_account_info, additional_options, config_path)
 
+    if sdk_marathon.app_exists(service_name):
+        log.info("%s index %s as %s is already installed", package_name, index, service_name)
+        log.info("Uninstalling %s index %s as %sd", package_name, index, service_name)
+        sdk_install.uninstall(package_name, service_name)
```
Comment (mpereira) [inline scale-tests/setup_streaming.py]: Ideally we wouldn't uninstall already running services, but instead just make sure that their information are in the `infrastructure.json` file.

**Comment by akirillov** [comment]
light ping @alexeygorobets and @mpereira: how shall we proceed with this PR?

**Comment by akirillov** [comment]
ping @alexeygorobets and @mpereira: how shall we proceed with this PR?

**Comment by mpereira** [comment]
Let's close this one for now.

**Pull request closed without merging**